site/src/examples/card/Variants.tsx

```diff
@@ -1,22 +1,44 @@
-import { Card, GridLayout } from "@salt-ds/core";
+import { Card, GridLayout, Panel, StackLayout } from "@salt-ds/core";
 import type { ReactElement } from "react";
 import styles from "./index.module.css";
 
 export const Variants = (): ReactElement => {
   return (
-    <GridLayout
-      columns="repeat(auto-fill, 260px)"
-      className={styles.variantContainer}
-    >
-      <Card variant="primary" className={styles.variantCard}>
-        Primary
-      </Card>
-      <Card variant="secondary" className={styles.variantCard}>
-        Secondary
-      </Card>
-      <Card variant="tertiary" className={styles.variantCard}>
-        Tertiary
-      </Card>
-    </GridLayout>
+    <StackLayout className={styles.variantContainer}>
+      <GridLayout
+        columns="repeat(auto-fill, 260px)"
+        className={styles.variantContainer}
+      >
+        <Card variant="primary" className={styles.variantCard}>
+          Primary
+        </Card>
+        <Card variant="secondary" className={styles.variantCard}>
+          Secondary
+        </Card>
+        <Card variant="tertiary" className={styles.variantCard}>
+          Tertiary
+        </Card>
+      </GridLayout>
+      <GridLayout
+        columns="repeat(auto-fill, 260px)"
+        className={styles.variantContainer}
+      >
+        <Panel variant="primary">
+          <Card variant="alpha" className={styles.alphaCard}>
+            Alpha
```

Comment:
General Question: Are the examples below, still considered Primary, Secondary, Tertiary tiles ?